src/vsr/replica.zig

```diff
@@ -1465,7 +1465,7 @@ pub fn Replica(
             // This saves us from going to disk. And we don't need to worry that the WAL's copy
             // of an uncommitted prepare is lost/corrupted.
             if (self.pipeline_prepare_for_op_and_checksum(op, checksum)) |prepare| {
-                log.debug("{}: on_request_prepare: op={} checksum={} reply from pipeline", .{
+                log.debug("{}: on_request_prepare: op={} checksum={any} reply from pipeline", .{
```

Comment:
Why are these changed? `checksum` is an integer, which should print just fine with `{}`.

Reply:
`checksum` is an optional. These errors only show with Zig master.

Reply:
Ah, ok, thanks!